server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/BaseQProfileWsAction.java

```diff
@@ -19,13 +19,14 @@
  */
 package org.sonar.server.qualityprofile.ws;
 
+import org.sonar.api.server.ws.Definable;
 import org.sonar.api.server.ws.RequestHandler;
 import org.sonar.api.server.ws.WebService;
 
 /**
  * Marker interface for quality profile related web service end points
  */
-public interface BaseQProfileWsAction extends RequestHandler {
+public interface BaseQProfileWsAction extends RequestHandler, Definable<WebService.NewController> {
```

Comment:
why not create a WsAction interface which extends RequestHandler and Definable<WebService.NewController> ? then, when creating a new marker interface action, we would extend WsAction only. WDYT ?